src/simulation/m_acoustic_src.fpp

```diff
@@ -171,6 +176,23 @@ contains
 
             num_points = source_spatials_num_points(ai) ! Use scalar to force firstprivate to prevent GPU bug
 
+            call random_number(phi_rn(1:num_broadband_freq))
+            ! Ensure all the ranks have the same random phase shift
+            call s_mpi_send_random_number(phi_rn)
+            sum_BB = 0d0
+
+            !$acc loop seq
```

Comment:
I thought we may have to use sequential loop because the variable `sum_BB` depends on the last iteration. 

Would this be better `!$acc loop reduction(+:sum_BB)` for this loop just like line 215 here `!$acc loop reduction(+:myRho,B_tait,small_gamma)`?